tools/java/org/unicode/cldr/tool/ChartDelta.java

```diff
@@ -1061,167 +1015,16 @@ private boolean isExtraSplit(String extraPath) {
         return false;
     }
 
-    public Splitter getValueSplitter(XPathParts pathPlain) {
+    private Splitter getValueSplitter(XPathParts pathPlain) {
```

Comment:
This method could be refactored away; only one line to method and only one caller in this file.

Reply:
OK, I'll do that.